src/navigation/components/NavigationItem/components/NavigationLink/NavigationLink.tsx

```diff
@@ -31,14 +31,20 @@ export const NavigationLink: React.FC<NavigationLinkProps> = (props) => {
 
     if (linkExtraProps?.target || !Link) {
         return (
-            <a href={url} title={text} className={classes} {...rest} {...linkExtraProps}>
+            <a
+                href={url}
+                title={urlTitle || text}
```

Comment:
why is there a fallback to the text in this place but not in all places?

Reply:
For backward compatibility I left fallback here. But I didn't do one on other places to avoid unexpected tooltips. Moreover, I expect that the link text and link title will not be the same